### ISSUE TITLE
Don’t render &gt; like codes for edit field values.

### ISSUE THREAD

**Issue by BrentAllsop** [the-canonizer/canonizer.2.0#645]
If you want to put something like this in a camp statement:
&lt;html&gt;
You can do that with regular html escapes like:
&amp;lt;html&amp;gt;

This get’s accepted, and renders correctly on the camp page.

The problem is when you go to edit a camp statement by selecting the button:

“Submit Statement Update Based On This”

Something is rendering the &amp;lt; back to “<” so you get
&lt;html&gt;
In the edit window.
It should remain the raw:
&amp;lt;html&amp;gt;
so you don't have to convert them all back to &amp;lt;, before you submit it again.

**Comment by BrentAllsop** [comment]
Let us include another related issue in #645.
When you lead a wiki text line with spaces or tabs, it turns off the wike rendering for that line.
The problem is each line of a consecutive set of lines is done individually.
This results in both too much space between the rendered lines and
if only one line is too long, only that line get's the scroll bar.

They should all be combined, and the entire set should have the scroll bar.

If you separate the lines, with a line with no leading spaces or tabs, then split them up.

In other words, if you want to imbed some html or code in a camp statement, for cutting and pasting, this is how it should be done.

**Comment by akuks** [comment]
@rupalichavan9860 Can you test this whenever you get some time

**Comment by rupalichavan9860** [comment]
@akuks @BrentAllsop I have tested and working fine now. I can see codes on edit page

![image](https://user-images.githubusercontent.com/40416610/147350089-8b1f9bc9-cff2-48b8-91cd-d55343b47f93.png)

@BrentAllsop Can you please confirm and close?

**Comment by rupalichavan9860** [comment]
Verified on production
